examples/kind/test/e2e-kind.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 readonly CT_VERSION=v2.2.0
 readonly KIND_VERSION=0.2.1
 readonly CLUSTER_NAME=chart-testing
-readonly K8S_VERSION=v1.13.4
+readonly K8S_VERSION=v1.14.1
```

Comment:
Can we make the same change in https://github.com/helm/chart-testing/blob/d9dba63311e5d0524832bf35e9c83993a76af93d/e2e-kind.sh?

Reply:
done

Reply:
done